bindings/protocols/coap/index.html

```diff
@@ -61,79 +61,130 @@ <h2>Protocol Overview</h2>
-            Observing resources as defined in [[RFC7641]], for example, can be enabled by including a
-            <code>subprotocol</code> value of <code>coap:observe</code> in a form.
-            Parameters for blockwise transfer (as defined in [[RFC7959]]) can be set in a
-            <code>coap:blockwise</code> member.
-            The vocabulary terms are defined in full detail in section [[[#vocabulary]]].
+            The section below gives an overview of the features offered by CoAP at the time of writing
+            and provides examples of a how they can be used in a [[[WOT-THING-DESCRIPTION]]].
+            The vocabulary terms used in these examples are defined in section [[[#vocabulary]]] below.
         </p>
         <!-- Maybe RFC numbers could be included in the table? -->
-        <table class="def">
-            <thead>
-                <tr>
-                    <th>Feature</th>
-                    <th>Covered by</th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr>
-                    <td>Request to Server</td>
-                    <td><code>"href": ..., "coap:method": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Request to Forward Proxy</td>
-                    <td><code>"href": ..., "coap:method": ..., "coap:proxy": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Caching: Validation</td>
-                    <td>n/a</td>
-                </tr>
-                <tr>
-                    <td>Conditional Requests</td>
-                    <td>n/a</td>
-                </tr>
-                <tr>
-                    <td>Content</td>
-                    <td><code>"coap:format": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Content Negotiation</td>
-                    <td><code>"coap:accept": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Observing Resources</td>
-                    <td><code>"subprotocol": "coap:observe"</code></td>
-                </tr>
-                <tr>
-                    <td>Block-Wise Transfer</td>
-                    <td><code>"coap:blockwise": { ... }</code></td>
-                </tr>
-                <tr>
-                    <td>Quick Block-Wise Transfer</td>
-                    <td><code>"coap:qblockwise": { ... }</code></td>
-                </tr>
-                <tr>
-                    <td>Hop Limit</td>
-                    <td><code>"coap:hoplimit": ...</code></td>
-                </tr>
-                <!--
-                <tr>
-                    <td>OSCORE</td>
-                    <td><code>SecurityScheme</code></td>
-                </tr>
-                <tr>
-                    <td>EDHOC</td>
-                    <td><code>SecurityScheme</code></td>
-                </tr>
-                -->
```

Comment:
With https://github.com/w3c/wot-thing-description/pull/1474 merged, we could actually also define these security schemes in this or an additional document. We could also define a security scheme for [ACE-OAuth](https://datatracker.ietf.org/doc/draft-ietf-ace-oauth-authz/) (which is very close to becoming RFC 9200) in the process.

Reply:
👍 I think this document would be a good starting point to describe these. It should probably be a separate pull request, though.